### ISSUE TITLE
LoadError: cannot load such file /logtail-rails-0.1.8/lib/logtail-rails/tasks/logtail.rake

### ISSUE THREAD

**Issue by leoplct** [logtail/logtail-ruby-rails#31]
I have installed as recommend in the getting started tutorial. 
I can successfully run rails console and send an event

```
rails console
Rails.logger.error("Something bad happened.")
```

but when I start the local webserver

```
bin/dev
=> Booting Puma
14:22:45 web.1  | => Rails 7.0.4.2 application starting in development 
14:22:45 web.1  | => Run `bin/rails server --help` for more startup options
14:22:45 css.1  | rails aborted!
14:22:45 css.1  | LoadError: cannot load such file -- /Users/myname/.rvm/gems/ruby-3.1.2/gems/logtail-rails-0.1.8/lib/logtail-rails/tasks/logtail.rake
14:22:45 css.1  | /Users/myname/.rvm/gems/ruby-3.1.2/gems/dead_end-4.0.0/lib/dead_end/core_ext.rb:65:in `load'
14:22:45 css.1  | /Users/myname/.rvm/gems/ruby-3.1.2/gems/dead_end-4.0.0/lib/dead_end/core_ext.rb:65:in `load'
14:22:45 css.1  | /Users/myname/.rvm/gems/ruby-3.1.2/gems/logtail-rails-0.1.8/lib/logtail-rails/railtie.rb:11:in `block in <class:Railtie>'
14:22:45 css.1  | /Users/myname/.rvm/gems/ruby-3.1.2/gems/railties-7.0.4.2/lib/rails/railtie.rb:286:in `instance_exec'
14:22:45 css.1  | /Users/myname/.rvm/gems/ruby-3.1.2/gems/railties-7.0.4.2/lib/rails/railtie.rb:286:in `block in run_tasks_blocks'
14:22:45 css.1  | /Users/myname/.rvm/gems/ruby-3.1.2/gems/railties-7.0.4.2/lib/rails/railtie.rb:298:in `each'
14:22:45 css.1  | /Users/myname/.rvm/gems/ruby-3.1.2/gems/railties-7.0.4.2/lib/rails/railtie.rb:298:in `each_registered_block'
14:22:45 css.1  | /Users/myname/.rvm/gems/ruby-3.1.2/gems/railties-7.0.4.2/lib/rails/railtie.rb:286:in `run_tasks_blocks'
14:22:45 css.1  | /Users/myname/.rvm/gems/ruby-3.1.2/gems/railties-7.0.4.2/lib/rails/application.rb:505:in `block in run_tasks_blocks'
14:22:45 css.1  | /Users/myname/.rvm/gems/ruby-3.1.2/gems/railties-7.0.4.2/lib/rails/engine/railties.rb:15:in `each'
14:22:45 css.1  | /Users/myname/.rvm/gems/ruby-3.1.2/gems/railties-7.0.4.2/lib/rails/engine/railties.rb:15:in `each'
14:22:45 css.1  | /Users/myname/.rvm/gems/ruby-3.1.2/gems/railties-7.0.4.2/lib/rails/application.rb:505:in `run_tasks_blocks'
14:22:45 css.1  | /Users/myname/.rvm/gems/ruby-3.1.2/gems/railties-7.0.4.2/lib/rails/engine.rb:464:in `load_tasks'
14:22:45 css.1  | /Users/myname/Dropbox/dev/amazon/Rakefile:6:in `<main>'
14:22:45 css.1  | /Users/myname/.rvm/gems/ruby-3.1.2/gems/railties-7.0.4.2/lib/rails/commands/rake/rake_command.rb:20:in `block in perform'
14:22:45 css.1  | /Users/myname/.rvm/gems/ruby-3.1.2/gems/railties-7.0.4.2/lib/rails/commands/rake/rake_command.rb:18:in `perform'
14:22:45 css.1  | /Users/myname/.rvm/gems/ruby-3.1.2/gems/railties-7.0.4.2/lib/rails/command.rb:51:in `invoke'
14:22:45 css.1  | /Users/myname/.rvm/gems/ruby-3.1.2/gems/railties-7.0.4.2/lib/rails/commands.rb:18:in `<main>'
14:22:45 css.1  | /Users/myname/.rvm/gems/ruby-3.1.2/gems/bootsnap-1.16.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
14:22:45 css.1  | /Users/myname/.rvm/gems/ruby-3.1.2/gems/bootsnap-1.16.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require'
14:22:45 css.1  | bin/rails:4:in `<main>'
```

**Gemfile**
`gem "logtail-rails"`

**Gemfile.lock**
```
    logtail (0.1.12)
      msgpack (~> 1.0)
    logtail-rack (0.2.4)
      logtail (~> 0.1)
      rack (>= 1.2, < 4.0)
    logtail-rails (0.1.8)
      actionpack (>= 3.0.0)
      activerecord (>= 3.0.0)
      logtail (~> 0.1)
      logtail-rack (~> 0.1)
      railties (>= 3.0.0)
```

config/application.rb

```
    config.logger = Logger.new(STDOUT)
    config.logger = Logtail::Logger.create_default_logger("XXXX")
```

**Comment by curusarn** [comment]
Hi @leoplct,

Thank you for raising this issue!
I'm passing this to the team to take a look and resolve this. I'll make sure to keep you updated here on GitHub.

Thanks again for reaching out.

**Comment by PetrHeinz** [comment]
Hello @leoplct, thanks for reaching out!

A possible cause of the issue it that you're using the latest version of `logtail` and `logtail-rack` gems, but an older version of `logtail-rails` (current is [`v0.2.6`](https://rubygems.org/gems/logtail-rails/versions/0.2.6)). I'm not seeing any obvious incompatibility with your Ruby or Rails versions.

Did you add the gem via `bundle add logtail-rails`, or did you add it to your `Gemfile` manually? Maybe you accidentally locked it on `v0.1`?

Could you please update the `logtail-rails` gem to `v0.2.6` and try again? 🙏

**Comment by PetrHeinz** [comment]
I've checked the source and this seems like an issue fixed in `0.2.2`, see https://github.com/logtail/logtail-ruby-rails/pull/21 for details.

Sorry about that 🙌 We've improved our CI processes since then, so similar issues shouldn't pop up in the future, hopefully.
